lms/urls.py

```diff
@@ -825,6 +825,11 @@
     url(r'^commerce/', include('commerce.urls', namespace='commerce')),
 )
 
+if settings.FEATURES.get('ENABLE_COURSE_GOALS'):
+    urlpatterns += (
+        url(r'^api/course_goal', include('lms.djangoapps.course_goals.urls', namespace='course_goals_api')),
```

Comment:
`api/course_goals`

Reply:
I see that api/grades was plural. It looked like the examples in the API doc all used singular for the API name. Maybe we just need more examples in the API doc, but it seems like we aren't consistent outside of the doc.

Reply:
the edX REST API Conventions page also says: 

* Resource name: 
   * Must: Use plural rather than singular nouns

Reply:
@jcdyer: Agreed. It's the difference between the API name (seen here) and the resource name, which comes later:
```
/api/course_goal/v0/course_goals
```